reference/6/Microsoft.PowerShell.Core/New-PSSessionConfigurationFile.md

```diff
@@ -315,11 +315,11 @@ AssembliesToLoad = 'System.Web.Services', 'FSharp.Compiler.CodeDom.dll'
 Adds the specified aliases to sessions that use the session configuration. Enter a hash table with
 the following keys:
 
-- Name - Name of the alias. This key is required.
-- Value - The command that the alias represents. This key is required.
-- Description - A text string that describes the alias. This key is optional.
+- Name - Name of the alias. This key is required
```

Comment:
These are complete sentences and more than one per list item. The period should be returned (many places in this doc).

Reply:
fixed